apps/site/src/components/NavBar/NavBar.module.scss

```diff
@@ -0,0 +1,40 @@
+@use "@/lib/styles/variables.scss" as variables;
+
+$red: #ff0000;
+
+.nav {
+	background-image: url("../../assets/images/lined_paper.svg");
+	background-size: cover;
+	background-position: left top;
+	background-repeat: no-repeat;
+}
+
+@media screen and (max-width: 576px) {
```

Comment:
Instead of hardcoding a value for `max-width`, you can use Bootstrap Sass mixins for added readability. See https://www.copycat.dev/blog/bootstrap-breakpoints/.